### DISCUSSION TITLE
[Suggestion]Enter Unixtime

### DISCUSSION THREAD

**Discussion by CRC32EX** [google/mozc#772] [Ideas]
As you know, We can enter current date & time.
|input|output|
|:--|:--|
|きょう|2023/06/15|
|いま|14:51|

So, I need new option.
|input|output|
|:--|:--|
|うにxtいめ<br>* typing `unixtime` in japenese mode|1686808345|

**Comment by ghost** [comment]
「ゆにっくすたいむ」で表示されるほうが自然だと思います。

**Comment by CRC32EX** [comment]
@avalanchemarramrat 
I agree natural is better.
But, I feel so tired (14 or 16) vs 8.

Only programmers use unixtime.
Visibility is not important.
Natural expressions is not required.

Typing everyday, so tired.

|Visible word|Typed keys|Typed key length|
|:--|:--|:--|
|ゆにっくすたいむ|yunikkusutaimu|14|
|ゆにっくすたいむ|yunixtukusutaimu|16|
|うにxtいめ|unixtime|8|

**Comment by CRC32EX** [comment]
As an alternative, implement replaceable dictionary.
```
{unixtime}
```

Like this image.
Everybody can enter anything you like.
![image](https://github.com/google/mozc/assets/25629617/065c20b3-3f87-49bc-abcc-ac6aff883511)

**Comment by ghost** [comment]
> @avalanchemarramrat I agree natural is better. But, I feel so tired (14 or 16) vs 8.
> 
> Only programmers use unixtime. Visibility is not important. Natural expressions is not required.
> 
> Typing everyday, so tired.
> Visible word 	Typed keys 	Typed key length
> ゆにっくすたいむ 	yunikkusutaimu 	14
> ゆにっくすたいむ 	yunixtukusutaimu 	16
> うにxtいめ 	unixtime 	8

その理屈だと、「今日」のローマ字表記は "kyô" ないし "kyō" なのですから、「きょ(kyo)」で今日の日付が表示されるべきである、って話になりません？かな漢字変換システムなのですから、英語からではなくかなから変換すべきではないでしょうか。

**Comment by ghost** [comment]
そもそも論として、ローマ字入力だけでなくかな入力も想定しなくてはならない、ということはあります。（Mozcが方針を変えてかな入力のサポートを打ち切らない限りは）

**Comment by hiroyuki-komatsu** [comment]
Hi CRC32EX,
Thank you for your request,

Would you tell us how and when you want to type a unixtime?
I'm wondering if typing a unixtime is a too advanced feature...

Thank you,
